dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/AndroidTvComponentPredicate.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.tracking
+
+import androidx.fragment.app.Fragment
+
+/**
+ * Special ComponentPredicate which can be used when defining tracking strategy for
+ * Android TV applications. By using this predicate we will exclude by default the extra
+ * view events created by `android.leanback` package helper fragments:
+ * `androidx.leanback.app.HeadersSupportFragment`, `androidx.leanback.app.RowsSupportFragment`.
+ * In case you also want to use your own implementation of a ComponentPredicate you can wrap
+ * it inside the `AndroidTvComponentPredicate`:
```

Comment:
```suggestion
 * Special ComponentPredicate which can be used when defining tracking strategy for
 * Android TV applications. By using this predicate, by default the extra
 * view events created by `android.leanback` package helper fragments are excluded:
 * `androidx.leanback.app.HeadersSupportFragment`, `androidx.leanback.app.RowsSupportFragment`.
 * In case you also want to use your own implementation of a ComponentPredicate you can wrap
 * it inside the `AndroidTvComponentPredicate`:
```